data/json/items/armor/suits_protection.json

```diff
@@ -844,9 +844,9 @@
     "looks_like": "touring_suit",
     "color": "dark_gray",
     "covers": [ "LEGS", "TORSO", "ARMS" ],
-    "coverage": 95,
-    "encumbrance": 12,
-    "max_encumbrance": 25,
+    "coverage": [ [ "leg_l", 95 ], [ "leg_r", 95 ], [ "torso", 95 ], [ "arm_l", 95 ], [ "arm_r", 95 ] ],
+    "encumbrance": [ [ "leg_l", 12 ], [ "leg_r", 12 ], [ "torso", 12 ], [ "arm_l", 12 ], [ "arm_r", 12 ] ],
+    "max_encumbrance": [ [ "leg_l", 25 ], [ "leg_r", 25 ], [ "torso", 25 ], [ "arm_l", 25 ], [ "arm_r", 25 ] ],
```

Comment:
This syntax is bad. try this type of syntax instead:
`
"coverage": 
  {
    "leg_l": 95,
    "leg_r": 95,
    "torso": 95
  }
`
and so on. i know there are instances of this syntax in the codebase, but that is legacy code... and we want the json to be human-readable, it's a must

Reply:
So a step further, what I was really looking for is something like this
"armor_data": [
  {
    "parts": [ "arm_l", "arm_r" ],
    "coverage": 25,
    "encumbrance": 5
  },
  {
    "parts": [ "torso", "leg_l", "leg_r" ],
    "coverage": 95,
    "encumbrance": [ 12, 25 ]
  }
]

Reply:
This is why I gave the garment/article description before, to get a more complete definition

Reply:
kevin's solution would be much more friendly toward unhardcoding bodyparts.